Documentation/developer-setup-linux.md

```diff
@@ -9,52 +9,67 @@ After compiling Unreal Engine, set the following environment variables in your `
-- `~/dev/cesium-unreal-samples/Plugins/cesium-unreal/extern/cesium-native` - The directory for the base libraries project.
+- `~/dev/cesium-unreal` - The directory for the actual *Cesium for Unreal* plugin.
+- `~/dev/cesium-unreal/extern/cesium-native` - The directory for the base libraries project.
 
-You may use any directory for the project, but the directory for the actual *Cesium for Unreal* plugin **MUST** be in a subdirectory `Plugins/cesium-unreal` of the project directory. This way, Unreal will automatically find the Plugin when running the project, and pick up any changes that have been made to the plugin.
+In this setup, we will build the Cesium for Unreal plugin separately from any project, and then install it as an Engine plugin.
 
 This can be set up with the following sequence of commands, on the console, starting in the `~/dev` directory:
 
     git clone https://github.com/CesiumGS/cesium-unreal-samples.git
-    cd cesium-unreal-samples
-    mkdir Plugins
-    cd Plugins
-    git clone --recursive https://github.com/CesiumGS/cesium-unreal.git
+    git clone -b ue4-main --recursive https://github.com/CesiumGS/cesium-unreal.git
 
 > Note: The last line will also check out the `cesium-native` submodule and its dependencies. If you forget the `--recursive` option, you will see many compiler errors later in this process. If this happens to you, run the following in the `Plugins\cesium-unreal` directory to update the submodules in the existing clone:
 
-     git submodule update --init --recursive
+    git submodule update --init --recursive
 
 # Building cesium-native
 
 The cesium-native libraries and their dependencies use CMake and must be built separately from Cesium for Unreal. There are a number of ways to do this, but typically on Linux this is done with CMake from the command-line.
 
 ## CMake command-line
 
-First, configure the CMake project in the `~/dev/cesium-unreal-samples/Plugins/cesium-unreal/extern` directory by following the instructions below.
+First, configure the CMake project in the `~/dev/cesium-unreal/extern` directory by following the instructions below.
 **Note**: The following steps must be done in the `extern` directory, and *not* the `cesium-native` subdirectory!
 
-Change to the `~/dev/cesium-unreal-samples/Plugins/cesium-unreal/extern` directory, and execute the following commands to build and install a Debug version of cesium-native:
+Change to the `~/dev/cesium-unreal/extern` directory, and execute the following commands to build and install a Debug version of cesium-native:
 
-      cmake -B build -S . -DCMAKE_TOOLCHAIN_FILE="unreal-linux-toolchain.cmake" -DCMAKE_POSITION_INDEPENDENT_CODE=ON -DCMAKE_BUILD_TYPE=Debug
-      cmake --build build --target install
+    cmake -B build -S . -DCMAKE_TOOLCHAIN_FILE="unreal-linux-toolchain.cmake" -DCMAKE_POSITION_INDEPENDENT_CODE=ON -DCMAKE_BUILD_TYPE=Debug
+    cmake --build build --target install -j14
```

Comment:
I may be wrong about this, but my belief was that `cmake --build build --target install` was essentially `-jMAX`. Would be be better to leave the `-j14` out since it would vary based on hardware?

Reply:
In my experience it's single-threaded without that option, but maybe that's only on Windows. I'll check.

Reply:
I just did a test. First I removed the build directory, configured, and built with `-j14`, and it took 57 seconds to do the build part. Then I did the same again without `-j14` and it took .. oh I don't know, it been a few minutes now and it's only at 59%. 

So I can believe newer versions of Clang (UE uses 11.0.1) or CMake (I have 3.18.4) changed to a smarter default, but it still seems worth mentioning. I'll change the wording, though.